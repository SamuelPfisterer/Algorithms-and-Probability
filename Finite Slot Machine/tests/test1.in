30
10 14
2 1 0 1 1 1 1 3 2 2
10 16
1 2 2 0 0 10 0 0 0 1
10 11
2 1 0 0 0 3 0 2 0 3
10 14
2 1 0 0 0 0 0 9 1 1
10 20
1 1 3 2 1 1 2 3 3 3
10 20
2 2 1 2 0 2 0 8 1 2
10 12
2 1 3 0 0 2 1 1 1 1
10 13
2 2 0 2 2 0 0 1 2 2
10 19
3 3 1 3 1 2 0 1 3 2
10 19
0 0 9 1 2 0 2 2 1 2
10 10
2 0 1 1 2 3 0 1 0 0
10 12
0 8 1 0 2 0 0 1 0 0
10 11
0 2 0 1 3 2 1 0 0 2
10 21
1 1 2 2 10 2 1 0 0 2
10 17
3 1 3 1 3 1 1 1 2 1
10 17
2 1 0 2 8 1 2 0 0 1
10 7
0 0 1 2 2 0 1 1 0 0
10 19
2 2 2 1 2 0 0 0 2 8
10 12
3 1 2 0 2 1 1 1 0 1
10 17
1 0 2 1 9 0 1 2 0 1
10 19
3 2 1 3 0 3 3 0 3 1
10 18
1 2 0 0 1 2 1 8 2 1
10 13
0 0 1 3 1 0 3 2 3 0
10 21
2 1 1 2 9 1 2 0 2 1
10 16
2 1 3 2 0 1 3 1 1 2
10 23
1 2 10 0 1 2 2 2 1 2
10 15
0 0 3 0 3 3 2 1 0 3
10 19
0 2 2 1 10 2 0 0 2 0
10 20
2 3 1 1 3 2 3 3 0 2
10 16
2 1 1 0 0 9 1 1 1 0
