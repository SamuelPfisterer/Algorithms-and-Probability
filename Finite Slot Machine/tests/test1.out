0.12223152
0.5452381
0.2494228
0.51148851
0.13392556
0.27698508
0.10147306
0.12839136
0.12833874
0.25674731
0.1923545
0.4543771
0.17200818
0.32317205
0.10456713
0.31558817
0.13809524
0.343177
0.09339226
0.36599212
0.15228528
0.30932863
0.21514458
0.264316
0.11906967
0.24128913
0.19259241
0.38721499
0.13306331
0.4025641
